src/app/views/interactives/InteractivesForm.jsx

```diff
@@ -274,7 +274,7 @@ export default function InteractivesForm(props) {
                                         type="button"
                                         buttonText="Preview"
                                         class="secondary"
-                                        onClick={() => console.log()}
+                                        onClick={() => window.open(`${rootPath}/interactives/show/${interactiveId}`)}
```

Comment:
there is react-router you can use in the repo

Reply:
done